eppo_core/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "eppo_core"
-version = "4.1.1"
+version = "4.2.0"
```

Comment:
Main change here was removing `reqwest::blocking` and introducing `tokio` for async handling. Do you think this helps/hurts any threading stability? Any additional improvements for handling exceptions in tokio?

Reply:
This should make no difference as we're just doing what `reqwest::blocking` did under the hood

Reply:
sorry, just found that we included breaking changes in 4.1.1, so we need a major bump: https://github.com/Eppo-exp/eppo-multiplatform/pull/107